cmd/root.go

```diff
@@ -5,41 +5,39 @@ import (
 	"os"
 
 	log "github.com/sirupsen/logrus"
+
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"github.com/vmware-tanzu/sonobuoy/cmd/sonobuoy/app"
 
 	"github.com/openshift/provider-certification-tool/pkg/assets"
-	"github.com/openshift/provider-certification-tool/pkg/client"
 	"github.com/openshift/provider-certification-tool/pkg/destroy"
 	"github.com/openshift/provider-certification-tool/pkg/retrieve"
 	"github.com/openshift/provider-certification-tool/pkg/run"
 	"github.com/openshift/provider-certification-tool/pkg/status"
 	"github.com/openshift/provider-certification-tool/pkg/version"
 )
 
-var ()
-
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "openshift-provider-cert",
 	Short: "OpenShift Provider Certification Tool",
 	Long:  `OpenShift Provider Certification Tool is used to evaluate an OpenShift installation on a provider or hardware is in conformance`,
-	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		var err error
 
 		// Validate logging level
 		loglevel := viper.GetString("loglevel")
 		logrusLevel, err := log.ParseLevel(loglevel)
 		if err != nil {
-			return err
+			log.Fatal(err)
 		}
 		log.SetLevel(logrusLevel)
 
-		// Save kubeconfig
-		client.Kubeconfig = viper.GetString("kubeconfig")
-
-		return nil
+		// Additional log options
+		customFormatter := new(log.TextFormatter)
+		customFormatter.FullTimestamp = true
+		log.SetFormatter(customFormatter)
```

Comment:
Unless you need to use the `customFormatter`, you could use the inline instantiation like:
```suggestion
		log.SetFormatter(&log.TextFormatter{
		    FullTimestamp: true,
		})
```